raiwidgets/raiwidgets/fairness_metric_calculation.py

```diff
@@ -13,6 +19,98 @@
     "needs to provide a MetricFrame class to calculate metrics. For an " \
     "refer to fairlearn.metrics.MetricFrame"
 
+z_score = 1.959964  # This z-score corresponds to the 95% confidence interval
+alpha = 0.95        # Conventional level of power for statistical tests
+digits_of_precision = 4
+
+
+def compute_wilson_bounds(p, n, digits=digits_of_precision, z=z_score):
```

Comment:
As discussed, please put in a few tests of these new functions you're adding, even if you just have to evaluate the functions with particular values, and then have a test that verifies (using `pytest.approx`) that the value is stable.